admin-frontend/src/components/AnalyticsPage.vue

```diff
@@ -1,5 +1,91 @@
-<template>Analytics placeholder</template>
+<template>
+  <PowerBIReportEmbed
+    v-if="isDashboardLoaded"
+    :embed-config="configSubmissionAnalytics"
+    :css-class-name="powerBiCssClass"
+  ></PowerBIReportEmbed>
+  <!-- <PowerBIReportEmbed
+    v-if="isDashboardLoaded"
+    :embed-config="configUserBehaviour"
+    :css-class-name="powerBiCssClass"
+  ></PowerBIReportEmbed>
+  <PowerBIReportEmbed
+    v-if="isDashboardLoaded"
+    :embed-config="configDataAnalytics"
+    :css-class-name="powerBiCssClass"
+  ></PowerBIReportEmbed> -->
+</template>
 
-<script lang="ts"></script>
+<script setup lang="ts">
+import { PowerBIReportEmbed } from 'powerbi-client-vue-js';
+import { models, IReportEmbedConfiguration } from 'powerbi-client';
+import { ref } from 'vue';
+import ApiService from '../services/apiService';
+import { ZonedDateTime, Duration } from '@js-joda/core';
+import { POWERBI_RESOURCE } from '../utils/constant';
 
-<style></style>
+const isDashboardLoaded = ref<boolean>(false);
+const powerBiCssClass = 'powerbi-container';
+
+// Bootstrap Dashboard by leaving some details undefined
+const configSubmissionAnalytics = ref<IReportEmbedConfiguration>({
+  type: 'report',
+  id: undefined,
+  embedUrl: undefined,
+  accessToken: undefined,
+  tokenType: models.TokenType.Embed,
+  hostname: 'https://app.powerbi.com',
```

Comment:
can we have this as a configurable value instead of hardcoded

Reply:
done